.github/workflows/yb-ctl-test.yml

```
@@ -8,13 +8,17 @@ on:
       - master
     paths:
       - '**/yb-ctl*'
+    paths-ignore:
+       - 'docs/**'
```

Need to adjust the syntax. Can't have `paths` and `paths-ignore` together.
```suggestion
      - '**/yb-ctl*'
      - '!docs/**'
```